### PR TITLE
chore: fix global PUT for task config policies

### PR DESCRIPTION
**Pull request by kkunapuli**
## Ticket

[CM-534]

## Description

Fix global PUT API for task config policies to include a parameter for the admin input.

## Test Plan

None needed.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CM-534]: https://hpe-aiatscale.atlassian.net/browse/CM-534?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: proto/src/determined/api/v1/config_policies.proto [hunk 1]
```diff
@@ -45,9 +57,14 @@ message PutGlobalConfigPoliciesResponse {
 // GetWorkspaceConfigPoliciesRequest lists task config policies
 // for a given workspace and workload type.
 message GetWorkspaceConfigPoliciesRequest {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "workload_type" ] }
```
Comment (amandavialva01) [inline proto/src/determined/api/v1/config_policies.proto:61]: is `workspace_id` also required here?

File: proto/src/determined/api/v1/config_policies.proto [hunk 2]
```diff
@@ -76,9 +97,14 @@ message GetGlobalConfigPoliciesResponse {
 // DeleteWorkspaceConfigPoliciesRequest is used to delete all task config
 // policies for the workspace and workload type.
 message DeleteWorkspaceConfigPoliciesRequest {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "workload_type" ] }
```
Comment (amandavialva01) [inline proto/src/determined/api/v1/config_policies.proto:101]:
```suggestion
    json_schema: { required: [ "workload_type", "workspace_id" ] }
```
(maybe this?)

**Review by amandavialva01 — COMMENTED** [review]
Looks great! Left a few comments on required protobuf fields

File: proto/src/determined/api/v1/config_policies.proto (hunk 1 above)
Comment (kkunapuli) [reply proto/src/determined/api/v1/config_policies.proto:61]: Great suggestion; I'll add it to all the workspace requests.

**Review by amandavialva01 — APPROVED** [review]
LGTM, great work!

**Pull request merged**